src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -2410,10 +2410,23 @@ private void OnAutoEvoPredictionComplete(PendingAutoEvoPrediction run)
         autoEvoPredictionRunSuccessful = true;
 
         // Set the initial value
-        totalPopulationLabel.ResetInitialValue();
-        totalPopulationLabel.Value = run.PlayerSpeciesOriginal.Population;
+        totalEnergyLabel.ResetInitialValue();
+        totalEnergyLabel.Value = run.PlayerSpeciesOriginal.Population;
 
-        totalPopulationLabel.Value = newPopulation;
+        results.GetPatchEnergyResults(run.PlayerSpeciesNew).TryGetValue(
+            Editor.CurrentPatch, out var newTotalPatchEnergyGathered);
+        var totalEnergyLabelText = totalEnergyLabel.GetNode<Label>(totalEnergyLabel.ValuePath);
+
+        if (newTotalPatchEnergyGathered != null)
+        {
+            totalEnergyLabel.Value = newTotalPatchEnergyGathered.TotalEnergyGathered;
+            if (totalEnergyLabelText != null)
+            {
+                totalEnergyLabelText.Text =
+                    @$"{StringUtils.ThreeDigitFormat(
+                        newTotalPatchEnergyGathered.TotalEnergyGathered)} ({newPopulation})";
+            }
+        }
 
         var sorted = results.GetPopulationInPatches(run.PlayerSpeciesNew).OrderByDescending(p => p.Value).ToList();
```

Comment:
It seems you didn't modify the best patch and worst patch numbers. They should also be updated to the new format (energy first and then in parentheses the population).

Reply:
Theoretically I made it work, but I don't know if it looks pretty enough:
image

Reply:
>  Edit: also predicted global population is lower than predicted local population, is that normal?

It *might* be a case of population number adjustment being kinder on the player, so the population result has that lower strength applied, but the per patch result might not.

It does look a bit busy so maybe just showing the energy numbers in the best and worst patches would be better? If I remember right that was my original idea I had about this, but I didn't recall that until just now.

Reply:
I guess they could be kept for future reference if they are easy to comment out (you should also comment that they used to be displayed but are now kept just for the potential future use they might have).